hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip869/NodeCreateTest.java

```diff
@@ -313,36 +309,18 @@ final Stream<DynamicTest> allFieldsSetHappyCaseForIps() throws CertificateEncodi
      */
     @LeakyEmbeddedHapiTest(
             reason = NEEDS_STATE_ACCESS,
-            overrides = {"nodes.gossipFqdnRestricted", "nodes.webProxyEndpointsEnabled"})
+            overrides = {"nodes.gossipFqdnRestricted"})
     final Stream<DynamicTest> allFieldsSetHappyCaseForDomains() throws CertificateEncodingException {
         final var nodeCreate = canonicalNodeCreate();
-        return hapiTest(
-                overridingTwo("nodes.gossipFqdnRestricted", "false", "nodes.webProxyEndpointsEnabled", "true"),
-                newKeyNamed(ED_25519_KEY).shape(KeyShape.ED25519),
-                nodeCreate,
-                verifyCanonicalCreate(nodeCreate),
-                viewNode("nodeCreate", node -> {
-                    assertEqualServiceEndpoints(GOSSIP_ENDPOINTS_FQDNS, node.gossipEndpoint());
-                    assertEqualServiceEndpoints(SERVICES_ENDPOINTS_FQDNS, node.serviceEndpoint());
-                    assertEqualServiceEndpoint(GRPC_PROXY_ENDPOINT_FQDN, node.grpcProxyEndpoint());
-                }));
-    }
-
-    @LeakyEmbeddedHapiTest(
-            reason = NEEDS_STATE_ACCESS,
-            overrides = {"nodes.gossipFqdnRestricted"})
-    final Stream<DynamicTest> allFieldsButProxyEndpointSet() throws CertificateEncodingException {
```

Comment:
Github shows the diff weirdly here. I deleted the `allFieldsButProxyEndpointSet()` test because it's not useful anymore.